src/components/OcBreadcrumb.vue

```diff
@@ -1,25 +1,31 @@
 <template>
-  <div :class="`oc-breadcrumb oc-breadcrumb-${variation}`">
-    <ul class="oc-breadcrumb-list">
+  <nav :class="`oc-breadcrumb oc-breadcrumb-${variation}`">
+    <ol class="oc-breadcrumb-list">
       <li v-for="(item, index) in items" :key="index" class="oc-breadcrumb-list-item">
-        <router-link
-          v-if="home && index === 0"
-          :to="item.to"
-          class="uk-flex uk-float-left"
-          :aria-label="homeAccessibleLabelValue"
+        <router-link v-if="item.to" tabindex="0" :to="item.to">{{ item.text }}</router-link>
+        <a
+          v-else-if="item.onClick"
+          tabindex="0"
```

Comment:
`tabindex="0"` is the default for an `a` tag. Should not be needed, as long as it has a non-empty `href`. For the `router-link` I'm not sure if it's needed (didn't check).

Reply:
Let me go again trough this, IHMO in this particular case the <a> with item.onClick never will have a href here

Reply:
According to the VUE docs, tabindex might not be necessary for router-link:
```
<router-link> is the component for enabling user navigation in a router-enabled app. The target location is specified with the to prop. It renders as an <a> tag with correct href by default
```

https://router.vuejs.org/api/#router-link

Reply:
Woah, didn't pay close enough attention. If it has a click handler it should not be an `a` tag. Could you check the usage in github.com/owncloud/web and github.com/owncloud/file-picker if a button would be semantically correct here?

Reply:
sounds suitable, wondering if the oc-button can also be aria-current=page?

Reply:
Should work. All attributes that you apply on a component will be forwarded to the root native component within the component template.

Reply:
@kulmann this needs a bit of css adjustments, should we dig into that?